app/views/users/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>This is the user#index view</h1>
\ No newline at end of file
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Users</title>
+</head>
+<body>
+<section>
+  <% @user.each do |user| %>
```

Comment:
Please make sure that the user index page should follow [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png) wireframe.
![posts](https://user-images.githubusercontent.com/35504149/190831347-c9e25152-566a-4c34-87f6-652455ee271d.png)